plugins/c8y_configuration_plugin/src/config.rs

```diff
@@ -75,53 +105,64 @@ mod tests {
     fn deserialize_plugin_config() {
         let config: PluginConfig = toml::from_str(
             r#"
-             files = [
-                '/etc/tedge/tedge.toml',
-                '/etc/tedge/mosquitto-conf/c8y-bridge.conf',
-                '/etc/tedge/mosquitto-conf/tedge-mosquitto.conf',
-                '/etc/mosquitto/mosquitto.conf'
-            ]"#,
+            [[files]]
+                path =  "/etc/tedge/tedge.toml"
+            [[files]]
+                path =  "/etc/tedge/mosquitto-conf/c8y-bridge.conf"
+            [[files]]
+                path =  "/etc/tedge/mosquitto-conf/tedge-mosquitto.conf"
+            [[files]]
+              path =  "/etc/mosquitto/mosquitto.conf"
+             "#,
```

Comment:
It would be good to add a test case with the alternative syntax. The point is more about internal documentation rather than testing.

```
files = [
  {path = "/etc/tedge/tedge.toml"}, 
  {path = "/etc/tedge/mosquitto-conf/tedge-mosquitto.conf"},
  {path = "/etc/mosquitto/mosquitto.conf"}
]
```